courses/free-courses-en.md

```diff
@@ -1108,8 +1108,12 @@
 
 ### Solidity
 
+* [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript](https://www.youtube.com/watch?v=gyMwXuJrbJQ) - Patrick Collins
 * [Learn Solidity](https://youtube.com/playlist?list=PL16WqdAj66SCOdL6XIFbke-XQg2GW_Avg) - Will it Scale (YouTube)
 * [MASTER Solidity for Blockchain](https://youtube.com/playlist?list=PLS5SEs8ZftgVnWHv2_mkvJjn5HBOkde3g) - Dapp University (YouTube)
+* [Solidity 101](https://www.youtube.com/watch?v=5eLqFac5Tkg&list=PLYORQHvGMg-WS5r8UjaWnnAeCHTH3wRaF) - Secureum
```

Comment:
What @eshellman meant was \

```suggestion
* [Solidity 101](https://www.youtube.com/playlist?list=PLYORQHvGMg-WS5r8UjaWnnAeCHTH3wRaF) - Secureum
```

Do the same thing for other entries also

In case of the line you mentioned, it's a standalone video, whereas this is a playlist

Reply:
@shrn01 Now can you check?